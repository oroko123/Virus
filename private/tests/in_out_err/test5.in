znaczek 1000 1910 Poczta Polska
znaczek 1000 1905 Poczta Polska
znaczek 1000 1906 Poczta Polska
znaczek 1000 1904 Poczta Polska
znaczek 1000 1907 Poczta Polska
znaczek 1000 1903 Poczta Polska
znaczek 1000 1908 Poczta Polska
znaczek 1000 1902 Poczta Polska
znaczek 1000 1909 Poczta Polska
znaczek 1000 1901 Poczta Polska
1900 1910
