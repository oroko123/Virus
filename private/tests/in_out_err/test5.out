1901 Poczta Polska 1000 znaczek
1902 Poczta Polska 1000 znaczek
1903 Poczta Polska 1000 znaczek
1904 Poczta Polska 1000 znaczek
1905 Poczta Polska 1000 znaczek
1906 Poczta Polska 1000 znaczek
1907 Poczta Polska 1000 znaczek
1908 Poczta Polska 1000 znaczek
1909 Poczta Polska 1000 znaczek
1910 Poczta Polska 1000 znaczek
